tests/Annotation/DITest.php

```diff
@@ -23,6 +23,7 @@ public function testYamlConvertion()
         $di->public = true;
         $di->abstract = false;
         $di->parent = '@test_parent_dependency';
+        $di->lazy = true;
```

Comment:
You should also add annotation to symfony-autodi-yaml/tests/Fixtures/Annotation/ComplexClass.php  and validate in https://github.com/metglobal-compass/symfony-autodi-yaml/blob/master/tests/Finder/AnnotationFinderTest.php#L57

Reply:
Added ComplexClass test